### DISCUSSION TITLE
Inconsistent loading of font

### DISCUSSION THREAD

**Discussion by Jaxattack** [ivodolenc/nuxt-font-loader#25] [Q&A]
In my production site the font is being picked up in inspector, and the CSS rules are there, but the font is rendering inconsistently. On page load/reload, there seems to be a 50% chance of the font either being rendered correctly or falling back to the sans-serif fallback declaration.

![Screenshot 2023-11-28 at 3 30 04 PM](https://github.com/ivodolenc/nuxt-font-loader/assets/4566235/62b0a061-30b4-40b5-a8d9-b1630245ae86)

**Comment by ivodolenc** [comment]
Hi, I can't reproduce this with the default [playground](https://github.com/ivodolenc/nuxt-font-loader/tree/main/playground) which works as expected.
